src/BloomExe/Edit/EditingView.cs

```diff
@@ -692,7 +694,8 @@ private void OnChangeImage(DomEventArgs ge)
 			var imageElement = GetImageNode(ge);
 			if (imageElement == null)
 				return;
-			string currentPath = imageElement.GetAttribute("src").Replace("%20", " ");
+			var src = imageElement.GetAttribute ("src");
+			string currentPath = HttpUtilityFromMono.UrlDecode(src);
 
```

Comment:
These changes crept in as part of a previous iteration of this pull request.  Since they're now totally unrelated to the actual bugfix, should they be put in a separate PR as general code cleanup/improvement?